### PR TITLE
Verbose

### PR DESCRIPTION
**Pull request by davebulaval**
Added a flag to trigger messages during embedding upload into the model.
Added a flag to trigger messages during the download of models weights and embeddings.

### CODE REVIEW

File: deepparse/fasttext_tools.py [hunk 1]
```diff
@@ -68,24 +67,26 @@ def download_fasttext_embeddings(lang_id: str, saving_dir: str) -> str:
...
     vectors from fastText's website https://fasttext.cc/docs/en/crawl-vectors.html and save it in the
     saving directory (saving_path).
     """
 
     url = "https://dl.fbaipublicfiles.com/fasttext/vectors-crawl/%s" % gz_file_name
-    warnings.warn("The fastText pre-trained word embeddings will be download (6.8 GO), "
-                  "this process will take several minutes.")
-    _download_file(url, saving_path)
+    if verbose:
+        print("The fastText pre-trained word embeddings will be download (6.8 GO), "
```
Comment (MAYAS3) [inline deepparse/fasttext_tools.py:79]: will be downloaded

File: deepparse/parser/address_parser.py [hunk 2]
```diff
@@ -53,6 +53,7 @@ class AddressParser:
 
             The default value is GPU with the index ``0`` if it exist, otherwise the value is ``CPU``.
         rounding (int): The rounding to use when asking the probability of the tags. The default value is 4 digits.
+        verbose (bool): Turn on/off the verbose of the model. The default value is True.
```
Comment (MAYAS3) [inline deepparse/parser/address_parser.py:56]: verbosity?

**Review by MAYAS3 — CHANGES_REQUESTED** [review]

File: deepparse/parser/address_parser.py [hunk 3]
```diff
@@ -156,6 +166,8 @@ def __call__(self,
         # since training data is lowercase
         lower_cased_addresses_to_parse = [address.lower() for address in addresses_to_parse]
 
+        if self.verbose and len(addresses_to_parse) > 64:
```
Comment (freud14) [inline deepparse/parser/address_parser.py:169]:
```python
and len(addresses_to_parse) > 64
```
Why?

Comment (davebulaval) [reply deepparse/parser/address_parser.py:169]: Yeah, it's not clear. It's the threshold where the time starts to be more of a concern.

Comment (davebulaval) [reply deepparse/parser/address_parser.py]: I've added a global variable with more information about it.
#dc3b619

**Review by freud14 — APPROVED** [review]

**Pull request merged**